src/utils.ts

```diff
@@ -269,7 +269,7 @@ export async function estimateTotalGasRequiredByUnsignedTransaction(
 
   // Find the total gas cost by taking the product of the gas price & the
   // estimated number of gas units needed.
-  return BigNumber.from(gasPrice).mul(gasTotalMultiplier).mul(estimatedGasUnits).toString();
+  return BigNumber.from(gasPrice).mul(gasTotalMultiplier).div(toBNWei(1)).mul(estimatedGasUnits).toString();
```

Comment:
Yeah. If you roll back the 2nd commit in this PR then the tests will fail with an ethers BigNumber underflow.

I hit exactly this issue in relayer-v2 last week and recognised that the implementation was similar to here, so I thought I'd head over and make a similar change.

It kinda annoys me that we promote it to 18 decimals...but it just seems consistent to always go to 18, instead of something bespoke.